tools/depends/configure.ac

```diff
@@ -101,8 +101,13 @@ AC_ARG_ENABLE([wayland],
 OLD_PATH="${PATH}"
 if test -n "$use_toolchain"; then
   PATH="${use_toolchain}/bin:${use_toolchain}/usr/bin:$PATH"
+  case $host in
+    *linux-android*)
+      AC_PROG_CXX([clang++ gcc g++])
+    ;;
+  esac
 fi
-AX_CXX_COMPILE_STDCXX_11(,[mandatory])
+AX_CXX_COMPILE_STDCXX_11([noext],[mandatory])
```

Comment:
@wsnipex as said in your PR, not sure about that one
Besides, it will impact all depends platforms, not only droid

Reply:
Actually we should only rely on pure C++11 standard and not any gnu extensions.
But I don't know if it will break anything.

Reply:
it shouldn't, but we'll see soon enough.

Reply:
Well, older gcc had shaky c++11 support.
If we go that route, we'd have to find out and set a minimum gcc, for no practical benefit as clang will use `-std=c++11` anyway (I assume)

Reply:
the min compiler versions are defined by the toolchains anyway, so a min version check for gcc is not strictly necessary, but could be done.
Clang seems to accept std=gnu++11 as well and c+11 is not standard.
>checking whether arm-linux-androideabi-clang++ accepts -g... yes
checking whether arm-linux-androideabi-clang++ supports C++11 features by default... no
checking whether arm-linux-androideabi-clang++ supports C++11 features with -std=gnu++11... yes

Reply:
ok